### DISCUSSION TITLE
SoftPatch Algo for Noisy Data NIPS 2022

### DISCUSSION THREAD

**Discussion by nicozhou7** [open-edge-platform/anomalib#794] [Feature Requests]
Hi, recently in NIPS 2022, there is a new paper called SoftPatch that can be better than Patchcore in Noisy Data. Any thoughts on this?
https://openreview.net/forum?id=pIYYJflkhZ

**Comment by samet-akcay** [comment]
@nixczhou, thanks for sharing this paper! I was not aware of this work, looks interesting!
